app/lib/screens/HomeScreens/ChatScreen.dart

```diff
@@ -118,6 +119,29 @@ class _ChatScreenState extends State<ChatScreen> {
     );
   }
 
+  Future<FfiBufferUint8> _userAvatar(String userId) async {
+    final member = await widget.room.getMember(userId);
+    return member.avatar();
+  }
+
+  Widget _avatarBuilder(String userId) {
```

Comment:
I think I have removed this as we don't need this custom method to apply avatars. The package already supports it and also have algorithm to change user avatar background colors based on ```userID```.  Or this is something we needed here?

Reply:
This function is needed to get avatar from user id.

Reply:
Then its okay to use. Another problem with this is that text avatars appear bigger than usual.

Reply:
Of course, I can add `width` & `height` parameters to `Member::avatar()` function.
But I think it is efficient to resize with Image class method on flutter.
Is it impossible to resize with Image class method on flutter?
We have to think about web or desktop too.